### DISCUSSION TITLE
Resize indicator with panel resize

### DISCUSSION THREAD

**Discussion by bertaveira** [foxglove/community#524] [Q&A]
When we have a lot of indicators they tend to be too big. From our experience 10 is the absolute maximum number of indicators in a 13 inch screen when the labels are not as small as "false". Would be nice for the text and "blob" to redimension with the panel size or have that option in the panel settings. That way the user could easily resize to their needs without worrying with cutting off the indicator

**Comment by jtbandes** [comment]
We can do some amount of auto-sizing, but properly sizing text to fit with CSS is kind of an impossible problem (see https://stackoverflow.com/questions/16056591/font-scaling-based-on-width-of-container)

What do you think about a setting in the sidebar to manually adjust the font size?

**Comment by bertaveira** [comment]
@jtbandes  That would be a very good alternative!
